Assets/MixedRealityToolkit.Extensions/SceneTransitionService/Scripts/CameraFaderQuad.cs

```diff
@@ -66,6 +66,15 @@ public async Task FadeOutAsync(float fadeOutTime, Color color, IEnumerable<Camer
                 try
                 {
                     quadMaterial = new Material(Shader.Find(QuadMaterialShaderName));
+                    // Set to fade
+                    quadMaterial.SetInt("_Mode", 2);
+                    quadMaterial.SetInt("_SrcBlend", (int)UnityEngine.Rendering.BlendMode.SrcAlpha);
+                    quadMaterial.SetInt("_DstBlend", (int)UnityEngine.Rendering.BlendMode.OneMinusSrcAlpha);
+                    quadMaterial.SetInt("_ZWrite", 0);
+                    quadMaterial.DisableKeyword("_ALPHATEST_ON");
```

Comment:
Does this work when you run on device? This keyword variants might not be built into the player build if they are not reference anywhere.

Reply:
Side note, I plan on adding some utility methods to avoid all this "boiler plate" when switching render modes.

Reply:
@Cameron Good question. It's been tested in Surfaces, but that project has other particle materials. Explicit references unfortunately aren't possible because of the way camera fader classes are used.

Reply:
After discussing with @Cameron-Micka we've updated the transition service profile to include 
an optional camera fader material. The camera fader interface also takes a reference to the transition service profile when initialized.